src/coreclr/src/gc/gcconfig.h

```diff
@@ -64,65 +64,71 @@ class GCConfigStringHolder
 // Each one of these keys produces a method on GCConfig with the name "Get{name}", where {name}
 // is the first parameter of the *_CONFIG macros below.
 #define GC_CONFIGURATION_KEYS \
```

Comment:
nit - looks like you changed many lines just to add whitespaces because the new names you added were longer - I don't actually think we need to do this. you can just make the new ones (that serve the same purpose) lined up. seems like there's no need to change all these white spaces for the rest.